### DISCUSSION TITLE
Is it possible to configure the character set for CFIND MWL requests

### DISCUSSION THREAD

**Discussion by clancien** [dcm4che/dcm4che#1276] [Q&A] [answered]
¿ Can i configure the character set for all C-FIND MWL request I receive from modalities ?

¿ And this can be configure it per Modality or only globally ?

TagCode: 0008,0005 

Thanks in advance

**Comment by gunterze** [comment, marked as answer]
DICOM does not define negotiation of accepted character sets. So a modality can't assume, that the Character Set specified in the identifier of the C-FIND RQ to query a MWL SCP - which is only required if it contains is a non-ascii matching key - matches the  Character Set specified in the identifier of the C-FIND RSP returned by the MWL SCP.
